packages/block-library/src/post-date/edit.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * WordPress dependencies
+ */
+import { useEntityProp, useEntityId } from '@wordpress/core-data';
+import { __experimentalGetSettings, dateI18n } from '@wordpress/date';
+import { RichText } from '@wordpress/block-editor';
+import { __ } from '@wordpress/i18n';
+
+function PostDateDisplay() {
+	const [ date ] = useEntityProp( 'postType', 'post', 'date' );
+	const settings = __experimentalGetSettings();
+
+	return date ? (
+		<RichText.Content
+			tagName="h6"
```

Comment:
What prompted this decision to use an `h6` element? For screen readers, this will cause unexpected reading of heading semantics. So you have:

```
Heading 1 Post Title
Text Published On
Heading 6 Date and time
```

Why not use the [<time>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time) element instead?

Reply:
It was an arbitrary choice to get something somewhat formatted rendered quickly.

>Why not use the element instead?

Which element?

Reply:
Whoops, the `<time>` element, and I meant to post it as a link to https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time

Reply:
Done 👍 

Thanks for the review!